test/contracts/OVM/execution/OVM_ExecutionManager/ovmREVERT.spec.ts

```diff
@@ -16,6 +16,9 @@ const test_ovmREVERT: TestDefinition = {
       messageRecord: {
         nuisanceGasLeft: OVM_TX_GAS_LIMIT,
       },
+      transactionRecord: {
+        ovmGasRefund: 0,
+      },
```

Comment:
OK, we fixed the tests, but I think we also need to manually verify that there is no way to reach a pre-state with non-zero `ovmGasRefund`. I've been walking through the relevant code paths, but I think we should discuss together.